src/AWS.Deploy.Recipes/DeploymentBundleDefinitions/Container.deploymentbundle

```diff
@@ -11,6 +11,16 @@
             "AdvancedSetting": true,
             "Updatable": true
         },
+        {
+            "Id": "DockerfilePath",
+            "Name": "Dockerfile Path",
+            "Description": "Specifies the absolute path to the Dockerfile as an alternative to the default location.",
+            "Type": "String",
+            "TypeHint": "DockerfilePath",
```

Comment:
Can we have the `TypeHint` be generic like `FilePath` and then put in the `TypeHintData` additional information for the type of file to use. (i.e. File extensions, File Type). That way the VS toolkit can make a generic type hint that adds a browse button for anything with a `FilePath` TypeHint and optional add some filtering mechanism.

Reply:
1. Moved to generic `FilePathCommand` typehint
2. Added `FilePathTypeHintData`, to provide VS Toolkit with common properties that they can pass into a `System.Windows.Forms.FileDialog` to customize the file picker.